src/combat.ts

```diff
@@ -356,6 +398,20 @@ export class Macro {
     return new this().if_(condition, ifTrue);
   }
 
+  /**
+   * Create a new macro with an "if !" statement.
+   * @param condition The BALLS condition for the if statement.
+   * @param ifFalse Continuation if the condition is false.
+   * @returns {Macro} This object itself.
+   */
+  static ifNot_<T extends Macro>(
+    this: Constructor<T>,
+    condition: Parameters<T["ifNot_"]>[0],
```

Comment:
Rather than using Parameters, can we name the type that we use in if and ifn't somewhere?

Reply:
Other than that, I'm into it.

Reply:
I'm not really that experienced in typescript to do that